### PR TITLE
[macOS] Pester tests invocation after install.

### PR DESCRIPTION
**Pull request by Darleev**
# Description
Add Pester tests invocation right after tool's installation

#### Related issue:
https://github.com/actions/virtual-environments-internal/issues/1601
## Check list
- [+] Related issue / work item is attached
- [+] Tests are written (if applicable)

### CODE REVIEW

File: images/macos/provision/core/pipx-packages.sh [hunk 1]
```diff
@@ -17,3 +18,7 @@ for package in $pipx_packages; do
         pipx install $package
     fi
 done
+
+invoke_tests "PipxPackages"
+
+
```
Comment (maxim-lobanov) [inline images/macos/provision/core/pipx-packages.sh:24]: nitpick: remove empty lines

File: images/macos/tests/RubyGem.Tests.ps1 [hunk 2]
```diff
@@ -0,0 +1,39 @@
+Import-Module "$PSScriptRoot/../helpers/Common.Helpers.psm1"
```
Comment (maxim-lobanov) [inline images/macos/tests/RubyGem.Tests.ps1:1]: could you please double check that we really need this import?
It is already imported in https://github.com/actions/virtual-environments/blob/main/images/macos/helpers/Tests.Helpers.psm1#L1 and should be required to import again?

File: images/macos/tests/PHP.Tests.ps1 [hunk 3]
```diff
@@ -0,0 +1,19 @@
+
+    
+Import-Module "$PSScriptRoot/../helpers/Common.Helpers.psm1"
```
Comment (maxim-lobanov) [inline images/macos/tests/PHP.Tests.ps1:3]: could you please double check that we really need this import?
It is already imported in https://github.com/actions/virtual-environments/blob/main/images/macos/helpers/Tests.Helpers.psm1#L1 and should be required to import again?

File: images/macos/tests/PHP.Tests.ps1 [hunk 4]
```diff
@@ -0,0 +1,19 @@
+
+
```
Comment (maxim-lobanov) [inline images/macos/tests/PHP.Tests.ps1:2]: nitpick: empty lines

Comment (Darleev) [reply images/macos/tests/PHP.Tests.ps1]: Thanks.Done.

File: images/macos/tests/PHP.Tests.ps1 (hunk 3 above)
Comment (Darleev) [reply images/macos/tests/PHP.Tests.ps1]: Thanks.Done.

File: images/macos/tests/RubyGem.Tests.ps1 (hunk 2 above)
Comment (Darleev) [reply images/macos/tests/RubyGem.Tests.ps1]: Thanks.Done.

File: images/macos/provision/core/pipx-packages.sh (hunk 1 above)
Comment (Darleev) [reply images/macos/provision/core/pipx-packages.sh]: Thanks.Done.

**Review by maxim-lobanov — APPROVED** [review]

**Review by dibir-magomedsaygitov — APPROVED** [review]

**Review by al-cheb — APPROVED** [review]

**Pull request merged**